core/src/sigverify_stage.rs

```diff
@@ -344,6 +351,13 @@ impl SigVerifyStage {
     pub fn join(self) -> thread::Result<()> {
         self.thread_hdl.join()
     }
+
+    fn count_discarded_packages(batches: &[PacketBatch]) -> usize {
```

Comment:
would probably be better to count the ones rejected in the signature verify function and then report that back from `verify_batches` since it's already going through the packets.

Reply:
This fix is core particular than the others so it is in a separate commit: https://github.com/KirillLykov/solana/pull/1/commits/5fe18afff5e36eb52f73eb5b888f8a0d980f0ae5